pipelines/nox.py

```diff
@@ -30,7 +30,7 @@
 from pipelines import config as _pipelines_config
 
 # Default sessions should be defined here
-_options.sessions = []
+_options.sessions = ["reformat-code", "pytest-all-features", "flake8", "slotscheck", "mypy", "verify-types"]
```

Comment:
Seems more compelling to run `pytest-all-features` by default rather than `pytest` and if install speed is a concern `reuse_venv=True` could help with that

Reply:
`reuse_venv` is True by default. The reason it was not put there when we split the pipelines was because of not installing packages that might not be installable in the specific environment (like the need for a compiler). Nowadays, that is mostly useless, mostly because we even install `mypy` with all speedups.